### PR TITLE
Migrated to app log-routing

### PR DESCRIPTION
**Pull request by xavi-**

### CODE REVIEW

File: ebs_snapshots/snapshot_manager.py [hunk 1]
```diff
@@ -28,7 +28,16 @@ def run(connection, volume_id, interval='daily', max_snapshots=0, name=''):
...
         volumes = connection.get_all_volumes([volume_id])
     except EC2ResponseError as error:
-        logging.error(kayvee.formatLog("ebs-snapshots", "error", "failed to connect to AWS", {"msg": error.message}))
+        logging.error(kayvee.formatLog("ebs-snapshots", "error", "failed to connect to AWS", {
+            "msg": error.message,
+            "_kvmeta": {
+                "type": "notifications",
+                "channel": "#oncall-infra",
+                "icon": ":camera_with_flash:",
+                "user": "ebs-snapshots",
```
Comment (nathanleiby) [inline ebs_snapshots/snapshot_manager.py]: 🙏 will be nice to have a clearer source for these errors

File: main.py [hunk 2]
```diff
@@ -8,5 +8,14 @@
...
     except Exception as e:
-        logging.error(kayvee.formatLog("ebs-snapshots", "error", "unknown exception", {"error": str(e)}))
+        logging.error(kayvee.formatLog("ebs-snapshots", "error", "unknown exception", {
+            "error": str(e),
+            "_kvmeta": {
+                "type": "notifications",
+                "channel": "#oncall-infra",
+                "icon": ":camera_with_flash:",
+                "user": "ebs-snapshots",
+                "message": "ERROR: " + str(e),
```
Comment (nathanleiby) [inline main.py]: nit: `Error` vs `ERROR`

**Comment by nathanleiby** [comment]
lgtm

File: main.py (hunk 2 above)
Comment (xavi-) [inline main.py]: Normalized in 69427d3595de36fb18333bd1e53374bcfb1aa7d8

**Pull request merged**